### ISSUE TITLE
Game finish error

### ISSUE THREAD

**Issue by Jokiblack** [Kyrluckechuck/TFT-Bot#107]
When the game ends, I'm left with 0 health points, and the bot doesn't turn off the game until all the battles are finished with 0 health and 1 victory. Is this error only on my end? It wastes a lot of time with this error.

**Comment by Kyrluckechuck** [comment]
Yep, this is something that does happen, though I haven't been able to confirm if this still counts as time played towards the event pass (I have a feeling it may), it's still not ideal and we should get that fixed at some point!

**Comment by SahilDave04** [comment]
Let me have a crack at it.
Please direct me to where the changes are needed.

**Comment by Kyrluckechuck** [comment]
Yep, #111 should resolve this!
